modules/standard/List.chpl

```diff
@@ -796,6 +797,29 @@ module List {
       return result;
     }
 
+    /*
+      Returns a reference to the last item in this list.
+
+      .. warning::
+
+        Calling this method on an empty list will cause the currently running
+        program to halt. If the `--fast` flag is used, no safety checks will
+        be performed.
+
+      :return: A reference to the last item in this list.
+      :rtype: `ref eltType`
+    */
+    proc ref last() ref{
```

Comment:
Let's go ahead and call `_parSafeLastCompilerWarning` as the first statement of this function.